scripts/test/bulkImportPerformance/system-test-instance.properties

```diff
@@ -0,0 +1,276 @@
+
+# The number of bytes to allocate to the Arrow working buffer, which is used for sorting and other sundry activities (arrow only) [256MB]
+sleeper.ingest.arrow.working.buffer.bytes=268435456
+
+# The number of bytes to allocate to the Arrow batch buffer, which is used to hold the records (arrow only) [1GB]
+sleeper.ingest.arrow.batch.buffer.bytes=1073741824
+
+# The maximum number of bytes to store on the local disk before uploading to the main Sleeper store (arrow only) [2GB]
+sleeper.ingest.arrow.max.local.store.bytes=2147483648
+
+# The number of records to write at once into an Arrow file in the local store. A single Arrow file will be written to
+# multiple times and so this parameter does not affect the final size of the Arrow file (arrow only) [1K]
+sleeper.ingest.arrow.max.single.write.to.file.records=1024
+
+# The implementation of the async S3 client to use for upload during ingest.
+# Valid values are 'java' or 'crt'. This determines the implementation of S3AsyncClient that gets used.
+# With 'java' it makes a single PutObject request for each file.
+# With 'crt' it uses the AWS Common Runtime (CRT) to make multipart uploads.
+# Note that the CRT option is recommended. Using the Java option may cause failures if any file is >5GB in size, and
+# will lead to the following warning:
+# "The provided S3AsyncClient is not an instance of S3CrtAsyncClient, and thus multipart upload/download feature is not
+# enabled and resumable file upload is not supported. To benefit from maximum throughput, consider using
+# S3AsyncClient.crtBuilder().build() instead."
+# (async partition file writer only)
+sleeper.ingest.async.client.type=crt
+
+# The part size in bytes to use for multipart uploads.
+# (CRT async ingest only) [128MB]
+sleeper.ingest.async.crt.part.size.bytes=134217728
+
+# The target throughput for multipart uploads, in GB/s. Determines how many parts should be uploaded simultaneously.
+# (CRT async ingest only)
+sleeper.ingest.async.crt.target.throughput.gbps=10
+
+# Flag to enable/disable storage of tracking information for ingest jobs and tasks.
+sleeper.ingest.status.store.enabled=true
+
+# The time to live in seconds for ingest job updates in the status store. Default is 1 week
+sleeper.ingest.job.status.ttl=604800
+
+# The time to live in seconds for ingest task updates in the status store. Default is 1 week
+sleeper.ingest.task.status.ttl=604800
+
+########################################################################################
+#                              System Test Properties                                  #
+########################################################################################
+
+# The ingest mode to write random data. This should be either 'direct' or 'queue'.
+# 'Direct' means that the data is written directly. 'Queue' means that the data
+# is written to a Parquet file and an ingest job is created and posted to the
+# ingest queue.
+sleeper.systemtest.ingest.mode=generate_only
```

Comment:
The documentation here needs updating as it implies that direct and queue are the only options, and doesn't describe the new option generate_only.